tools/common/lib.go

```diff
@@ -73,27 +73,57 @@ const (
 	ConsumerGroupType = "CG"
 	// MinUnconsumedMessagesRetentionForMultiZoneDest is the minimum unconsumed retention allowed
 	MinUnconsumedMessagesRetentionForMultiZoneDest = 3 * 24 * 3600
+	// MinConsumedMessagesRetention is the minimum consumed retention
+	MinConsumedMessagesRetention = 60
+	// MaxConsumedMessagesRetention is the maximum consumed retention
+	MaxConsumedMessagesRetention = 100 * 24 * 3600
+	// MinUnconsumedMessagesRetention is the minimum unconsumed retention
+	MinUnconsumedMessagesRetention = 60
+	// MaxUnconsumedMessagesRetention is the maximum unconsumed retention
+	MaxUnconsumedMessagesRetention = 100 * 24 * 3600
+	// MinLockTimeoutSeconds is the minimum lock timeout seconds
+	MinLockTimeoutSeconds = 0
+	// MaxLockTimeoutSeconds is the maximum lock timeout seconds
+	MaxLockTimeoutSeconds = 100 * 24 * 3600
```

Comment:
This would be too long. I think generally lock timeout is in seconds. Maybe 1hr is more than enough